File: matilda/commands/protobuf/empty_converter.py
from google.protobuf.any_pb2 import Any
from google.protobuf.empty_pb2 import Empty

from matilda.commands.protobuf.protobuf_converter import ProtobufConverter


class EmptyConverter(ProtobufConverter[None]):
    def to_protobuf(self, value: None) -> Empty:
        return Empty()

    def from_protobuf(self, message: Any) -> None:
        return None

File: agent/commands-generator-api/src/main/java/org/matilda/commands/types/EmptyConverter.kt
package org.matilda.commands.types

import pbandk.wkt.Any
import pbandk.wkt.Empty

class EmptyConverter : ProtobufConverter<Void?> {
    override fun convertToProtobuf(obj: Void?):  Empty {
        return Empty()
    }

    override fun convertFromProtobuf(obj: Any): Void? {
        return null
    }
}
